controllers/managedocs_controller_test.go

```diff
@@ -777,6 +785,17 @@ var _ = Describe("ManagedOCS controller", func() {
 					Expect(secret.Data["key"]).Should(Equal([]byte("-----BEGIN PUBLIC KEY-----\ntest-validation-key\n-----END PUBLIC KEY-----")))
 				})
 			})
+			When("it is a provider deployment", func() {
+				It("should have rook-config-override configmap settings", func() {
+					if testReconciler.DeploymentType != providerDeploymentType {
+						Skip(fmt.Sprintf("Skipping the test as it is not required by %v", testReconciler.DeploymentType))
+					}
+					configMap := rookOverrideConfigMapTemplate.DeepCopy()
+					utils.WaitForResource(k8sClient, ctx, configMap, timeout, interval)
+					fmt.Println(configMap.Data["config"])
+					Expect(configMap.Data["config"]).Should(Equal("[global]\nosd_pool_pg_autoscale_mode = off\nosd_pool_default_pg_num = 128\nosd_pool_default_pgp_num = 128"))
```

Comment:
Same reason as the last comment:
```suggestion
					Expect(configMap.Data["config"]).Should(Equal("" +
						"[global]\n" +
						"osd_pool_pg_autoscale_mode = off\n" +
						"osd_pool_default_pg_num = 128\n" +
						"osd_pool_default_pgp_num = 128\n"))
```

Reply:
Done